halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java

```diff
@@ -62,16 +64,44 @@ private void safeExecuteThis() {
       executeThis();
     } catch (RetrofitError e) {
       if (e.getCause() instanceof ConnectException) {
-        AnsiUi.failure(e.getCause().getMessage());
+        AnsiUi.error(e.getCause().getMessage());
         AnsiUi.remediation("Is your daemon running?");
       } else {
-        HalconfigFixableIssue issue = (HalconfigFixableIssue) e.getBodyAs(HalconfigFixableIssue.class);
-        for (String warning : issue.getWarnings()) {
-          AnsiUi.warning(warning);
-        }
+        HalconfigProblemSet problemSet = (HalconfigProblemSet) e.getBodyAs(HalconfigProblemSet.class);
+
+        problemSet.sortIncreasingSeverity();
+        for (HalconfigProblem problem : problemSet.getProblems()) {
+          HalconfigProblem.Severity severity = problem.getSeverity();
+          HalconfigCoordinates coordinates = problem.getCoordinates();
+          String problemLocation;
+          String message = problem.getMessage();
+          String remediation = problem.getRemediation();
+
+          if (coordinates != null) {
+            problemLocation = "In " + coordinates + ":";
+          } else {
+            problemLocation = "Global:";
+          }
+
+          if (severity == HalconfigProblem.Severity.FATAL) {
```

Comment:
Ah nice, intellij was complaining about non-constant values in a case statement which from a compiler standpoint didn't make sense (enum values should be determined statically). Let me switch this around